src/libraries/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs

```diff
@@ -903,7 +904,9 @@ public void WorkingDirectory_Set_GetReturnsExpected(string workingDirectory)
             Assert.Equal(workingDirectory ?? string.Empty, info.WorkingDirectory);
         }
 
-        [Fact(Skip = "Manual test")]
```

Comment:
For true manual tests, we prefer `[ConditionalFact(nameof(ManualTestsEnabled))]` I assume? This came up yesterday.

Where I've seen
```
        internal static bool RunManualTests { get; } =
            !string.IsNullOrEmpty(Environment.GetEnvironmentVariable("CRYPTOGRAPHY_MANUAL_TESTS"));
```
and 
```c#
        public static bool ManualTestsEnabled => !string.IsNullOrEmpty(Environment.GetEnvironmentVariable("MANUAL_TESTS"));
```

Reply:
Yes.  It'd be nice if we could standardize on that and add something akin to the stress modes flag on the test environment type, such that we have one way to turn on manual tests and one property to check for it, rather than each suite coming up with its own.

Reply:
Right, although I don't think we'd ever want to "run all manual tests" since they are manual for different reasons eg destructive, need special setup, etc.

Reply:
> although I don't think we'd ever want to "run all manual tests"

No, but the same is true for stress.  I'm simply suggesting we should have a common approach and mechanism to marking tests as being manual and running them.  The common case will then be that someone does so when iterating on one library's tests.

Reply:
Yep. Probably, ` public static bool ManualTestsEnabled => !string.IsNullOrEmpty(Environment.GetEnvironmentVariable("MANUAL_TESTS"));` should go into `PlatformDetection` and then one just iterates on the library of choice.

Reply:
I'd have said https://github.com/dotnet/runtime/blob/d0dc66bc0b17e74a2ad5e9424d1cdfe6f3cb3869/src/libraries/Common/tests/TestUtilities/System/TestEnvironment.cs, but yeah, something like that.